components/options_resolver.rst

```diff
@@ -842,6 +842,29 @@ the option::
 This closure receives as argument the value of the option after validating it
 and before normalizing it when the option is being resolved.
 
+Ignore not defined Options
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, all options are resolved, validated and an :class:`Symfony\\Component\\OptionsResolver\\Exception\\UndefinedOptionsException`
+is thrown if an unknown option is passed. You can ignore not defined options by using the
+:method:`Symfony\\Component\\OptionsResolver\\OptionsResolver::ignoreUndefined` method::
+
+    // ...
+    $resolver
+        ->setDefined(['hostname'])
+        ->ignoreUndefined(true)
```

Comment:
It should be `setIgnoreUndefined(true)` here. Only the fluent configurator has this `ignoreUndefined()` method

Reply:
Oops, fixed !